CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertsPropertyResponse.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CautionaryAlertsApi.V1.Boundary.Response
+{
+    public class CautionaryAlertsPropertyResponse
+    {
+        public string PropertyReference { get; set; }
+        public string AddressNumber { get; set; }
```

Comment:
Do we not want to return UPRN here as well?

Reply:
Added UPRN to response